python/lsst/afw/geom/polygon/polygonLib.py

```diff
@@ -1,4 +1,6 @@
 from __future__ import absolute_import
 
+from lsst.afw.table.io import Persistable
+from ._polygon import PersistableFacadePolygon
```

Comment:
Why add this line? The next line pulls it in along with everything else in `_polygon`. I suggest deleting this (but keeping the line above, of course).